lib/components/PlayerScreen/queue_source_helper.dart

```diff
@@ -64,6 +64,7 @@ void navigateToSource(BuildContext context, QueueItemSource source) {
     case QueueItemSourceType.trackMix:
     case QueueItemSourceType.filteredList:
     case QueueItemSourceType.downloads:
+    case QueueItemSourceType.remoteClient:
```

Comment:
I don't think we'll ever implement a source target for this. So let's move it further up and add a `break`, like for `nextUp` or `unknown`. Right now you put it into the fall-through case which creates a "Not yet implemented" snackbar...